addons/create-cluster-admin/create-cluster-admin.addon

```diff
@@ -0,0 +1,5 @@
+# Name: create-cluster-admin
+# Description: Create admin user and add cluster-admin role to it.
+
+oc create user admin --full-name=admin
```

Comment:
@praveenkumar I suggest the user name should be "cluster-admin". I think thats what others agreed also @hferentschik @budhrg